ui/app/actions.js

```diff
@@ -215,7 +215,7 @@ function confirmSeedWords () {
     dispatch(actions.showLoadingIndication())
     log.debug(`background.clearSeedWordCache`)
     background.clearSeedWordCache((err, account) => {
-      dispatch(actions.hideLoadingIndication())
+      // dispatch(actions.hideLoadingIndication())
```

Comment:
Do we really want to comment this out?  What if there is not an error?  Does `showAccountDetail` also hide loading indication?